rec_to_nwb/processing/builder/originators/position_originator.py

```diff
@@ -53,14 +53,24 @@ def make(self, nwb_content: NWBFile):
                 position_df = self.get_position_with_corrected_timestamps(
                     position_tracking_path[0]
                 )
-                position.create_spatial_series(
-                    name=f"series_{dataset_ind}",
-                    description=", ".join(position_df.columns.tolist()),
-                    data=np.asarray(position_df),
-                    conversion=conversion,
-                    reference_frame="Upper left corner of video frame",
-                    timestamps=np.asarray(position_df.index),
-                )
+                #Multi-position split. 
+                #TODO: generalize key names?
+                key_lists = [['xloc','yloc','zloc'],
```

Comment:
I think we could just have:
```python
[['xloc','yloc'], # LED 1
['xloc2','yloc2']] # LED 2
```
since we will probably never generalize this code (especially in `rec_to_nwb`)

Reply:
Should LED indexing start from 1 or 0?

Reply:
Oh I don't think it really matters.